backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -243,13 +223,13 @@ public static Entity.Property UpdateProjectProperty(this Entity.ProjectProperty
                     if (originalProperty.Parcel == null || property.Parcel == null) throw new InvalidOperationException("Unable to transfer parcel.");
                     originalProperty.Parcel.AgencyId = property.Parcel.AgencyId;
                     originalProperty.Parcel.ClassificationId = property.Parcel.ClassificationId;
-                    originalProperty.Parcel.ProjectNumber = null;
+                    originalProperty.Parcel.ProjectNumbers = "[]";
```

Comment:
yeah, so I'm not 100% sure if this is correct. This logic is called when a property is transferred, so I'm not sure if removing all current project associations (as I'm doing here) makes sense, or if I should just remove from the current project.